lib/ladybug/energy_model/setpoint_humidistat_abridged.rb

```diff
@@ -0,0 +1,83 @@
+# *******************************************************************************
+# Ladybug Tools Energy Model Schema, Copyright (c) 2019, Alliance for Sustainable
+# Energy, LLC, Ladybug Tools LLC and other contributors. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# (1) Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# (2) Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'ladybug/energy_model/extension'
+require 'ladybug/energy_model/model_object'
+
+module Ladybug
+  module EnergyModel
+    class SetpointHumidistatAbridged < ModelObject
+      attr_reader :errors, :warnings
+  
+      def initialize(hash = {})
+        super(hash)
+        raise "Incorrect model type '#{@type}'" unless @type == 'SetpointAbridged'
+      end
+    
+      def defaults
+        result = {}
+        result
+      end
+    
+      def find_existing_openstudio_object(openstudio_model)
+        model_setpoint_humidistat = openstudio_model.getZoneControlHumistatByName(@hash[:name])
```

Comment:
You really shouldn't have a case where you use a method like this since each Zone/Space needs to get its own humidistat (it's not like people or lighting where you can use the same definition all over the place).

Reply:
I'm not sure I understand your question.  Are you asking whether the schema allows the assignment of a People definition to a Room that isn't used in any of the ProgramTypes? If so the answer is yes, very much so.  The primary reason why the schema allows you to specify a custom object for a Room is to handle cases where just one or two Rooms are slightly different from the others. This way, you don't need to create a whole new ProgramType schema just for those Rooms.